db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_02_08_001228) do
+ActiveRecord::Schema.define(version: 2023_01_31_202855) do
```

Comment:
why revert the schema?

Reply:
i dont see a migration here. I dont know how the schema got rolled forward but the number seems wrong